shared/common-adapters/dumb.desktop.js

```diff
@@ -40,6 +41,20 @@ const checkboxMap: DumbComponentMap<Checkbox> = {
   },
 }
 
+const iconMap: DumbComponentMap<Icon> = {
```

Comment:
generated icon sheet, just making one-per. we could discuss making something different but likely this'll make diffs easier to understand